fractal-modlets/Exercise2/problem/src/messyCode/Widget1.tsx

```diff
@@ -0,0 +1,16 @@
+import SomeGraph1 from "./SomeGraph1";
+import SomeControlPanel1 from "./SomeControlPanel1";
+
+import styles from "./Widget1.module.css"
+
+const Widget1 = () => {
+  return (
+    <div>
```

Comment:
Would be cool to see something other than `div` throughout the code (maybe `section`)?